src/operation-poller.ts

```diff
@@ -10,6 +10,7 @@ export interface OperationPollerOptions {
   backoff?: number;
   maxBackoff?: number;
   masterTimeout?: number;
+  once?: boolean;
```

Comment:
Did you mean to add this? If not please revert the change to this file.

Reply:
+1